packages/components/inputs/rich-text-input/src/rich-text-input.stories.tsx

```diff
@@ -107,7 +108,7 @@ export const PlaygroundExample: Story = () => {
         value={value}
       />
       <Text.Headline as="h3">Output</Text.Headline>
-      <pre style={{ textWrap: 'auto' }}>{value}</pre>
+      <code>{value}</code>
```

Comment:
The linter was complaining about the `auto` value of `text-wrap` (it's not a valid value) and since we only wanted the monospace-font and not the preserving of white-space, I made a `code` tag out of it.